doc/semidp.xml

```diff
@@ -0,0 +1,58 @@
+#############################################################################
+##
+#W  semidp.xml
+#Y  Copyright (C) 2017                                     Wilf A. Wilson
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+<#GAPDoc Label="DirectProduct">
+<ManSection>
+  <Func Name = "DirectProduct" Arg = "S[, T, ...]"/>
+  <Oper Name = "DirectProductOp" Arg = "list, S"/>
+  <Returns>A transformation semigroup.</Returns>
+  <Description>
+    The function <C>DirectProduct</C> takes an arbitrary positive number of
+    finite semigroups, and returns a semigroup that is isomorphic to their
+    direct product. <P/>
+
+    If these finite semigroups are either all partial perm semigroups, or all
+    bipartition semigroups, or all PBR semigroups, then <C>DirectProduct</C>
```

Comment:
Also if they are transformation semigroups, no? Also remove "either" since there are more than two possibilities, and also remove the first "or"

Reply:
It's true that if the args are all transformation semigroups, then the result is a transformation semigroup. But that falls under the next sentence: `Otherwise, <C>DirectProduct</C> returns a transformation semigroup`.

Reply:
Right you are, I'll merge this just now